telosnetwork/teloscan
# #531 | Chain overview redesign

# Fixes #531 

## Description
This PR adds in a chain overview section to the homepage as well as:
- adds stylized search bar to home
- tweaks the background color on light mode to be slightly greyish-purple for more contrast with tables
- removes unused assets
- changes light mode sun icon from fontawesome to google (FA one looks like a gear)

## Test scenarios
- go to https://deploy-preview-554--teloscan-redesign.netlify.app/
    - you should see a redesigned search bar on the home page
    - you should see a chain overview section on the home page
- resize the screen
    - the chain overview and search should look good at all screen sizes

## Checklist:
